app/models.py

```diff
@@ -967,19 +967,28 @@ def clarification_questions_are_closed(self):
     def status(self):
         if self.published_at is None:
             return 'draft'
+        elif self.withdrawn_at:
+            return 'withdrawn'
         elif self.applications_closed_at > datetime.utcnow():
             return 'live'
         else:
             return 'closed'
 
     @status.setter
     def status(self, value):
-        if value == self.status:
+        if self.status == value:
             return
+        elif self.status == 'withdrawn':
```

Comment:
I think if we look at the possible status changes that we allow there are only two:

1. draft to live
2. live to withdrawn

(Live to draft is no longer valid since it was decided in the kickoff to withdraw everything. Closed and withdrawn briefs can't change status. Draft to withdrawn is pointless. No status can be changed to 'closed'.)

With that in mind, I think the best way to simplify this logic is to explicitly cover the two valid cases:

```python
# Removing the short-circuit broke some tests in the past
# might be worth checking if we can get rid of it now
if value == self.status:
    return

if value == 'live' and self.status == 'draft':
    self.published_at = datetime.utcnow()
elif value == 'withdrawn' and self.status == 'live':
    self.withdrawn_at = datetime.utcnow()
else:
    raise ValidationError("Cannot change brief status from '{}' to '{}'".format(self.status, value))
```